content/news/playtest-20160403.md

```diff
@@ -0,0 +1,95 @@
+We are pleased to announce the first playtest leading up to our next stable release.
+
+Our development branch currently contains more than five months worth of changes, many of them very fundamental. As such, we have finally entered the feature freeze phase of the development cycle, where we stop adding features and focus on fixing bugs found during playtesting. This is where *you* come into play! We need *your* help to give this playtest as wide a test coverage as possible to ensure another successful stable release of OpenRA! We therefore ask you to please report any and all bugs and problems you encounter on [our bug tracker](http://bugs.openra.net). Thank you for helping to improve OpenRA!
+
+While there are the usual bug fixes and improvements, the biggest changes this time around are under the hood and will be felt foremost by server operators, modders and mappers. In fact, some of these changes are so fundamental that we will address them in-depth in the articles covering future playtests. To ensure that there will be enough dedicated servers available for playtesting, we will cover that topic below.
+
+Just to whet your appetite, dedicated servers no longer need to sync maps with the resource site, and it is now possible to add third-party mods as a single package (".oramod") to your installation.
+
+But one thing after the other...
```

Comment:
There's a bit of duplication across these paragraphs, i'll see if I can come up with a compressed version.